### ISSUE TITLE
Two announcements played simultaneously

### ISSUE THREAD

**Issue by jaoord** [flybywiresim/a32nx#6824]
### Aircraft Version

Development

### Build info

```json
{
  "built": "2022-02-28T23:00:03+00:00",
  "ref": "master",
  "sha": "d790b97b42fedc3d99a281f8d682ec77f183aaf6",
  "actor": "aguther",
  "event_name": "manual",
  "pretty_release_name": "master:d790b97b"
}
```

### Describe the bug

I was flying the following route:

EDDS/07 VESID2H VESID Y127 UBEGA Z818 RINEX DCT GULKO GULKO14 EDDK/14L

While descending from FL220 (cruise alt) to FL120 somewhere around FL200 two announcements where played simultaneously

### Expected behavior

No two announcements are played simultaneously

### Steps to reproduce

Load route as above.
Descend.

**Comment by patmack14** [comment]
what two announcements were played?

**Comment by Andrenilin** [comment]
2 announcements happen when you simultaneously reach cruise and then decent immediately. The announcement for TOC begins and then also the TOD announcement happens... all at the same time... Im not sure there is a fix for this as you are doing both things at once

**Comment by jaoord** [comment]
@patmack14 Exactly what two it were I cannot remember. It was also hard to distinguish one from the other.

@Andrenilin I had already been at cruise altitude for a while. Although I now realize according to the flightplan I should have been at FL240, but I went to FL220. 

I am now trying to recreate the flight and see if I can reproduce the behavior. 

Ideally I think, if two sounds are two be played simultaneously the should be queued so they are played in sequel. And when one of them is no longer applicable, it should be removed from the queue.

**Comment by jaoord** [comment]
Here's a video of the issue occurring again: https://youtu.be/r8seg7xZAgk

So, flight plan cruise was FL240, I set FL220 and after I've been at that altitude for a while I selected a lower altitude. I could not hear what messages were spoken

**Comment by hotshotp** [comment]
Right yeah it happens because you go from climb to descent phase and plays the cruise and the descent at the same time. I got a way to fix this and will do so in the next update 😃

**Comment by patmack14** [comment]
Are you thinking have a timer maybe and if not at the some TOC for a certain amount of time to not play anything?

**Comment by hotshotp** [comment]
Nah it's a bit different, don't worry, I already tested it and it works

**Comment by hotshotp** [comment]
Fixed via #6864
